lowdefy.yaml

```diff
@@ -244,9 +244,12 @@ pages:
               - id: save_survey
                 type: GoogleSheetAppendOne
                 connectionId: survey_sheet
-                properties:
```

Comment:
I think we should split the page into `pages/survey.yaml`. It can remain a single page, but just having one yaml file is not a best practice and a bad example

Reply:
Change made in e50942ac69c6ff1c65ab37413aa735a64ea16eb6